Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -2052,14 +2040,14 @@ void gotoConfirmToPrint() {
 
   void writeEEPROM() {
     dwinDrawStatusLine(GET_TEXT_F(MSG_STORE_EEPROM));
+    safe_delay(500);
     dwinUpdateLCD();
     DONE_BUZZ(settings.save());
   }
 
   void readEEPROM() {
-    const bool success = settings.load();
     dwinRedrawScreen();
-    DONE_BUZZ(success);
+    DONE_BUZZ(settings.load());
```

Comment:
`settings.load` should come ahead of directly redrawing the screen, because otherwise any screen that displays or uses settings will be displaying / using old settings.